src/pages/challenges.tsx

```diff
@@ -0,0 +1,57 @@
+import { useState } from 'react';
+import Layout from '../components/Layout';
+import Section from '../components/section';
+import Navbar from '../components/navbar';
+import Title from '../components/title';
+import ChallengesJson from '../../mock/challenges.json';
+
+const Challenges = (props) => {
+  const [state, setState] = useState({
+    showModal: false,
+  });
+
+  const btnClickHandler = () => {
+    const showModalVal = state.showModal;
+    setState({ showModal: !showModalVal });
+  };
+
+  const closeModal = () => {
+    setState({ showModal: false });
+  };
+
```

Comment:
Let's move the modal code to separate PR please @sumitd94

Reply:
@sumitd94 Are you not reading the review comments?

Reply:
Removed the code related to modal
Red : https://github.com/Real-Dev-Squad/website-roadmap/pull/8/commits/6c51e8168baa5f75e481493c92f0deb48db92c9c